src/ga/operators/crossover.rs

```diff
@@ -356,3 +358,100 @@ where
     (child_1, child_2)
   }
 }
+
+/// # Ordered crossover operator
+///
+/// This struct implements [self::CrossoverOperator] trait and can be used with GA.
```

Comment:
```suggestion
/// This struct implements [CrossoverOperator] trait and can be used with GA.
```

I know that I used this prefix (I thought it was necessary), but later I saw [our docs](https://docs.rs/ecrs/0.1.0-alpha.0/ecrs/ga/operators/crossover/struct.SinglePoint.html) and:

1. It is redundant
2. Looks bad